src/OceanSeaIceModels/ocean_sea_ice_model.jl

```diff
@@ -59,17 +61,37 @@ prettytime(model::OSIM)             = prettytime(model.clock.time)
 iteration(model::OSIM)              = model.clock.iteration
 timestepper(::OSIM)                 = nothing
 default_included_properties(::OSIM) = tuple()
-prognostic_fields(cm::OSIM)         = nothing
+prognostic_fields(::OSIM)           = nothing
 fields(::OSIM)                      = NamedTuple()
 default_clock(TT)                   = Oceananigans.TimeSteppers.Clock{TT}(0, 0, 1)
+time(model::OSIM)                   = model.clock.time
+checkpointer_address(::OSIM)        = "HydrostaticFreeSurfaceModel"
+
+reset!(model::OSIM) = reset!(model.ocean)
+
+initialize!(model::OSIM) = initialize!(model.ocean)
+
+initialize_jld2_file!(filepath, init, jld2_kw, including, outputs, model::OSIM) =
+    initialize_jld2_file!(filepath, init, jld2_kw, including, outputs, model.ocean.model)
+
+write_output!(c::Checkpointer, model::OSIM) = write_output!(c, model.ocean.model)
 
-function reset!(model::OSIM)
-    reset!(model.ocean)
+function set_clock!(sim)
```

Comment:
this _only_ works for coupled simulation

Reply:
[f9087aa](https://github.com/CliMA/ClimaOcean.jl/pull/401/commits/f9087aad4437c4d8322a5d5a3d292a5a63f0b8ec) creates a `set_clock!` for each simulation